.github/workflows/deploy.yml

```
@@ -0,0 +1,102 @@
+        uses: actions/upload-artifact@v4
+        with:
+          name: lambda-jar
+          path: build/libs/app.jar
+
+  deploy:
+    needs: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        stage: [dev, prod]
+    if: |
```

actionlint (1.7.8)

context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)